samples/bluetooth/mesh_demo/README.rst

```diff
@@ -0,0 +1,54 @@
+.. _ble_mesh_demo:
+
+Bluetooth: Mesh Demo
+####################
+
+Overview
+********
+
+A Bluetooth Mesh application primarily intended for demonstration
```

Comment:
Change to a sentence:

    This application is a simple demonstration of Bluetooth mesh functionality.

Reply:
I think that has a slightly different meaning though. What I want to highlight is that while most sample applications are just that (samples), the primary intention of this app is not to be sample code, rather to be something that when *used* acts as a simple mesh demonstration. I.e. the code itself (when viewed) is not a necessarily a good demonstration of mesh functionality.

Reply:
Ah, OK.  How about:

    This sample is a Bluetooth Mesh application intended
    for demonstration purposes only.

Reply:
sounds good